unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/BuildModeInWorldLoadingAudioHandler.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BuildModeInWorldLoadingAudioHandler : MonoBehaviour
+{
+    [SerializeField]
+    BuilderInWorldLoadingView loadingView;
+
+    private void Start() {
+        loadingView.OnHide += OnLoadingViewHide;
+    }
+
+    private void OnDestroy() {
+        loadingView.OnHide -= OnLoadingViewHide;
+    }
+
+    private void OnEnable() {
+        AudioScriptableObjects.builderEnter.Play();
+    }
+
+    private void OnLoadingViewHide() {
+        AudioScriptableObjects.builderReady.Play();
+    }
```

Comment:
We can move this to the BuilderInWorldLoadingView directly, so we don't have another class to handle this

Reply:
Fixed